stardist/bioimageio_utils.py

```diff
@@ -453,4 +453,13 @@ def import_bioimageio(source, outpath):
 
     model_class = (StarDist2D if config['n_dim'] == 2 else StarDist3D)
     model = model_class(None, outpath.name, basedir=str(outpath.parent))
+
+    # update the location of the stardist specific files, since they are the ones being used by the model
+    biomodel.config['stardist']['config'] = outpath / 'config.json'
+    biomodel.config['stardist']['thresholds'] = outpath / 'thresholds.json'
+    biomodel.config['stardist']['weights'] = outpath / weights
```

Comment:
Why is this helpful?

Reply:
E.g. to have the correct weight path that the stardist model is using, for example in the case where weights are updated.

Reply:
Sorry, I don't get it. If you want the preserve the config, thresholds, and weights at the time when the bioimage.io model was imported, then why should `biomodel.config['stardist']['config']` and `biomodel.config['stardist']['thresholds']` be overwritten?